### PR TITLE
hamming: update to new spec

### PR DESCRIPTION
**Pull request by wolf99**
Spec is updated to add two new tests that check for empty input strings

### CODE REVIEW

File: exercises/practice/hamming/.meta/tests.toml [hunk 1]
```diff
@@ -24,5 +24,11 @@
...
 "5dce058b-28d4-4ca7-aa64-adfe4e17784c" = false
 
+# disallow left empty strand
+"db92e77e-7c72-499d-8fe6-9354d2bfd504" = true
+
 # disallow right empty strand
 "38826d4b-16fb-4639-ac3e-ba027dec8b5f" = false
+
+# disallow left empty strand
+"920cd6e3-18f4-4143-b6b8-74270bb8f8a3" = true
```
Comment (ryanplusplus) [inline exercises/practice/hamming/.meta/tests.toml:34]:
```suggestion
"920cd6e3-18f4-4143-b6b8-74270bb8f8a3" = true

```

**Review by ryanplusplus — APPROVED** [review]
